src/features/MoviePage/Banner/styled.js

```diff
@@ -17,54 +25,136 @@ export const Shadow = styled.img`
 `;
 
 export const Poster = styled.img`
-  max-width: 1368px;
-  max-height: 770px;
-  margin: 0 auto;
-  display: flex;
+  max-width: 1356px;
   z-index: 2;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.laptop}) {
+    width: 100%;
+  }
 `;
 
 export const TitleWrapper = styled.div`
   color: ${({ theme }) => theme.color.white};
-  bottom: 56px;
-  left: 276px;
+  bottom: 0px;
+  left: 0px;
   z-index: 3;
   position: absolute;
   text-align: left;
-  display: flex;
-  flex-direction: column;
+  padding-bottom: 56px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.laptop}){
+padding-bottom: 10px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+    padding-bottom: 0px;
+    
+  }
 `;
 
 export const Title = styled.div`
   font-size: 64px;
   font-weight: 600;
-  line-height: 77px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.laptop}) {
+    font-size: 58px;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoint.tablet}) {
+    font-size: 48px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+    font-size: 34px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.largeMobile}) {
+    font-size: 30px;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoint.smallMobile}) {
+    font-size: 24px;
+  }
 `;
 
 export const Rating = styled.div`
+  max-width: 130px;
   display: flex;
   align-items: center;
-  gap: 8px;
-  margin-top: -20px;
+  flex-wrap: wrap;
+  margin: 20px 0;
+  gap: 6px;
+  @media (max-width: ${({ theme }) => theme.breakpoint.tablet}) {
+    margin: 5px 0;
+    max-width: 100%;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+  }
 `;
 
 export const StarImage = styled.img`
   width: 40px;
   height: 40px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.laptop}) {
+    width: 32px;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoint.tablet}) {
+    width: 26px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+    width: 22px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.smallMobile}) {
+    width: 16px;
+  }
 `;
 
 export const Rate = styled.p`
   font-size: 30px;
   font-weight: 500;
-  line-height: 39px;
+
+  margin: 0;
+  @media (max-width: ${({ theme }) => theme.breakpoint.laptop}) {
+    font-size: 25px;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoint.tablet}) {
+    font-size: 20px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobile}) {
+    font-size: 17px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.smallMobile}) {
+    font-size: 14px;
+  }
 `;
 
 export const RateMax = styled.p`
   font-size: 16px;
```

Comment:
I would add margin: 0 here

Reply:
If i gave margin: 0; than whole title goes down, and looks like not pretty. I would left it as it was, thanks.

Reply:
I used margin, and you're right, became better, thanks.